src/app/util/mock/Constants.h

```diff
@@ -50,5 +50,13 @@ constexpr EventId MockEventId(const uint16_t & id)
 {
     return (0xFFF1'0000 | id);
 }
+
+extern const uint16_t mockClusterRevision;
```

Comment:
These do not seem like constants. Having these in `Constants.h` seems odd.

Reply:
They are constant values returned by mock cluster attributes. I think it makes more sense to declare them as constants in the header that’s to repeat the literal values in the mock code and the test expectations — at least this way it’s easier to understand why the values match / the tests pass.

Reply:
Added some comments to them.